corona-warn-app/cwa-app-android
# Validation Result Common passed (EXPOSUREAPP-8031)

This PR addresses ExposureApp-8031 (valid result screen).

Description coming...

how to test:
- Scan a valid TestCert
- Click "Gültigkeit prüfen" inside the cert view
- check the result (failed, valid, etc.)
Note: as it is development state, you can set  _get() = State.PASSED_ (line 24) inside the _DCCValidation.kt_ file to force this screen.


Screenshots:

![vaild2](https://user-images.githubusercontent.com/46747780/125077325-42dcab00-e0c1-11eb-8259-aab2547b52c5.png)
(scrolled a little bit down)
![valid1](https://user-images.githubusercontent.com/46747780/125077328-43754180-e0c1-11eb-9473-95aee1d91e6f.png)
![dark1](https://user-images.githubusercontent.com/46747780/125081661-8b4a9780-e0c6-11eb-8d61-61b3d970a8f2.png)



- [x] Missing Darkmode elements
